addons/point_of_sale/static/tests/pos/tours/utils/ticket_screen_util.js

```diff
@@ -9,10 +9,21 @@ export function clickDiscard() {
         run: "click",
     };
 }
+/**
+ * Selects an order based on its sequence number.
+ * @param {string} orderName - It's recommended to pass the order sequence number 4-digited.
+ * This function ensures the sequence number is 4 digits by prepending leading zeros if necessary,
+ * to avoid any ambiguity with the session ID.
+ **/
 export function selectOrder(orderName) {
+    // Pad the order sequence with leading zeros to ensure it's 4 digits, preventing conflicts with session_id.
+    while (orderName.length < 4) {
+        orderName = "0" + orderName;
+    }
+
     return [
         {
-            trigger: `.ticket-screen .order-row:contains("${orderName}")`,
+            trigger: `.ticket-screen .order-row > *:nth-child(2):contains("${orderName}")`,
```

Comment:
You can maybe add a class on `order.getName()` in ticket_screen.js to be sure to target the right line. we've already encountered a few errors with this.